docs/comparison.md

```diff
@@ -1,42 +1,237 @@
+| Update mechanism                               | package manager (?)      | OTA<br>*controlled by Mycroft*             | git pull                  |
+| **Screen - GUI**                               |                          |                                            |                           |
+| GUI supported<br>*Show GUI if screen attached* | N/A                      | X11                                        | N/A                       |
+
+## Plugins
+
+### OpenVoiceOS
+
+|                         | ovos-buildroot                                    | ovos-arch                                         | ovos-raspbian                                     |
+|-------------------------|---------------------------------------------------|---------------------------------------------------|---------------------------------------------------|
+| **Default Plugins**     |                                                   |                                                   |                                                   |
+| VAD                     | ovos-vad-plugin-webrtcvad                         | ovos-vad-plugin-webrtcvad                         | ovos-vad-plugin-webrtcvad                         |
+| stt                     | ovos-stt-plugin-server<br>(ovos proxy)            | ovos-stt-plugin-server<br>(ovos proxy)            | ovos-stt-plugin-server<br>(ovos proxy)            |
+| fallback_stt            | N/A                                               | N/A                                               | N/A                                               |
+| tts                     | ovos-tts-plugin-mimic3-server<br>(public servers) | ovos-tts-plugin-mimic3-server<br>(public servers) | ovos-tts-plugin-mimic3-server<br>(public servers) |
+| fallback_tts            | ovos-tts-plugin-mimic                             | ovos-tts-plugin-mimic                             | ovos-tts-plugin-mimic                             |
+| **Recommended Plugins** |                                                   |                                                   |                                                   |
+| STT - On device         | ovos-stt-plugin-vosk                              | ovos-stt-plugin-vosk                              | ovos-stt-plugin-vosk                              |
+| STT - On premises       | ovos-stt-plugin-server<br>(any plugin)            | ovos-stt-plugin-server<br>(any plugin)            | ovos-stt-plugin-server<br>(any plugin)            |
+| STT - Cloud             | ovos-stt-plugin-server<br>(ovos proxy)            | ovos-stt-plugin-server<br>(ovos proxy)            | ovos-stt-plugin-server<br>(ovos proxy)            |
+| TTS - On device         | ovos-tts-plugin-mimic3                            | ovos-tts-plugin-mimic3                            | ovos-tts-plugin-mimic3                            |
```

Comment:
Same as above about lack of power

Reply:
should test this one, but i think that mimic3 should run in a pi3 too